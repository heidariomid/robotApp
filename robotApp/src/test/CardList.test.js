import React from 'react';
import {shallow} from 'enzyme';
import CardList from '../components/CardList';

describe('<CardList />', () => {
	const robotsMock = [{id: 2, name: 'test', email: 'test@yahoo.com'}];
	it('renders components', () => {
		const wrapper = shallow(<CardList robots={robotsMock} />);
		expect(wrapper.debug()).toMatchSnapshot();
	});
});
