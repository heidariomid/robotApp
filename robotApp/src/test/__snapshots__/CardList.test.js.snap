// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardList /> renders components 1`] = `
"<div>
  <Card id={2} name=\\"test\\" email=\\"test@yahoo.com\\" />
</div>"
`;
